client/src/components/Predictions.jsx

```diff
@@ -21,9 +22,15 @@ const Prediction = (props) => {
 const Predictions = (props) => {
   return props.predictions.map((prediction, i) => {
     return (
+<<<<<<< HEAD
```

Comment:
merge leftovers...

Reply:
Geez! This PR is sooo bad. lol I'm so sorry!!!